ratis-server/src/test/java/org/apache/ratis/InstallSnapshotFromLeaderTests.java

```diff
@@ -68,6 +68,7 @@
         prop, SNAPSHOT_TRIGGER_THRESHOLD);
     RaftServerConfigKeys.Snapshot.setAutoTriggerEnabled(prop, true);
     RaftServerConfigKeys.Log.Appender.setSnapshotChunkSizeMax(prop, SizeInBytes.ONE_KB);
+    RaftServerConfigKeys.Log.Appender.setRetryPolicy(getProperties(), "1ms,1000");
```

Comment:
@sumitagrawl  Why setting 1ms in tests?  Setting it so short essentially disable the feature.  Why not using the default?

Reply:
Updated, added only in one testcase where it is having wait time less for cluster to come up.